README.md

```diff
@@ -1,2 +1,87 @@
+
+
+||Tools 🛠️||
+|-|-------------|-|
+||![Git](https://img.shields.io/badge/git-%23F05033.svg?style=for-the-badge&logo=git&logoColor=white) ![ESLint](https://img.shields.io/badge/ESLint-4B3263?style=for-the-badge&logo=eslint&logoColor=white) ![GitHub](https://img.shields.io/badge/github-%23121011.svg?style=for-the-badge&logo=github&logoColor=white)   ![Markdown](https://img.shields.io/badge/markdown-%23000000.svg?style=for-the-badge&logo=markdown&logoColor=white)  ![Visual Studio Code](https://img.shields.io/badge/Visual%20Studio%20Code-0078d7.svg?style=for-the-badge&logo=visual-studio-code&logoColor=white) ![webpack](https://www.vectorlogo.zone/logos/js_webpack/js_webpack-ar21.svg)||
+
+## Getting Started 🏁
+
+To get a local copy up and running follow these simple example steps.
+
+> Clone the app by running this command
+
+```bash 
+git clone https://github.com/fobadara/gcxchange.git
+```
+> Make sure you install the dependencies by runing this command
+
+```bash
+npm install
+```
+
+
+> View it in your browser by running this command
+
+```bash
+ npm start
+```
+It runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000).
+
+**Prerequisites 📜**
+
+You will be needing:
+
+• A web browser to view output
+
+• A text editor or terminal
+
+**Run tests**
+
+You can run tests by following these steps:
+
+• commit and push your changes
+
+• navigate to pull request
+
+• I have created a pull request which runs linters or you could simply create a pull request of your own
+
+| Author ✍️ |
+|---|
+|👤 Abdulfatai Badara
+<a target="_blank" href="https://github.com/fobadara"><img src="https://img.shields.io/badge/github-%23121011.svg?style=for-the-badge&logo=github&logoColor=white" alt="Github profile"></a>  <a target="_blank" href="https://www.linkedin.com/in/fob90s"><img src="https://img.shields.io/badge/-LinkedIn-0077b5?style=for-the-badge&logo=LinkedIn&logoColor=white" alt="Linkedin profile"></a> <a target="_blank" href="https://twitter.com/fob90s"><img src="https://img.shields.io/badge/-Twitter-1DA1F2?style=for-the-badge&logo=Twitter&logoColor=white" alt="Twitter profile"></a>  
+<a target="_blank" href="mailto:fob90s@gmail.com"><img src="https://img.shields.io/badge/-Gmail-D14836?style=for-the-badge&logo=Gmail&logoColor=white" alt="Gmail account"></a> <a target="_blank" href="https://wa.me/+2349066478370"> <img src="https://img.shields.io/badge/WhatsApp-25D366?style=for-the-badge&logo=whatsapp&logoColor=white" alt="Whatsapp account"></a> (Click to reach me)|
+
+
+**🤝 Contributing**
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+**Show your support 💪**
+
+Give a ⭐️ if you like this project!
+
+Hat tip to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) for the design idea.
+
```

Comment:
- [OPTIONAL] Good job mentioning the design's author. However, you are acknowledging the author. Therefore, your readme should have an "Acknowledgements" section where you can specify the author.

Example:
```md
##Acknowledgements

[Nelson Sakwa](https://www.behance.net/sakwadesignstudio) for the design idea.

```

Reply:
Thank you. I will make sure I adjust my README file